.circleci/integration-tests/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM quay.io/astronomer/astro-runtime:4.2.4-base
+FROM quay.io/astronomer/ap-airflow:2.2.4
```

Comment:
for now, I'm reverting this since runtime image as well as ap-airflow:2.2.5 is not getting healthy after deployment